src/styles/main.scss

```diff
@@ -1,3 +1,6 @@
+@import "../styles/blocks/header.scss";
+@import "../styles/blocks/card.scss";
```

Comment:
You should make `git add <path to your folder>` to commit and push that files to